apps/api/src/models/ApplicationData.py

```diff
@@ -47,15 +47,15 @@ class BaseApplicationData(BaseModel):
     is_first_hackathon: bool
     linkedin: NullableHttpUrl = None
     portfolio: NullableHttpUrl = None
-    frq_change: Union[str, None] = Field(None, max_length=2048)
+    frq_change: str = Field(max_length=2048)
     frq_video_game: str = Field(max_length=2048)
 
 
 class BaseMentorApplicationData(BaseModel):
     model_config = ConfigDict(str_strip_whitespace=True, str_max_length=254)
 
-    experienced_technologies: Optional[List[str]] = []
-    pronouns: Optional[List[str]] = []
+    experienced_technologies: List[str] = []
+    pronouns: List[str] = []
```

Comment:
Chore: since we are on Python 3.9+, we can directly use generics of built-in containers, e.g. `list[str]` instead of using `typing.List`.